src/display/svg.js

```diff
@@ -1001,6 +1005,90 @@ SVGGraphics = (function SVGGraphicsClosure() {
       current.setCurrentPoint(x, y);
     },
 
+    rectangle: function SVGGraphics_rectangle(x, y, width, height) {
+      var current = this.current;
+
+      if (current.path) {
```

Comment:
Personally not happy about the checks in every function. Maybe we could create `current.path` at the start and ensure that it exists.